table/block_based_table_reader.h

```diff
@@ -115,6 +116,9 @@ class BlockBasedTable : public TableReader {
   InternalIterator* NewRangeTombstoneIterator(
       const ReadOptions& read_options) override;
 
+  InternalIterator* NewUnfragmentedRangeTombstoneIterator(
```

Comment:
should it be private?

Reply:
Yeah it should be; thanks for catching that!